tests/test_bfd_global.rb

```diff
@@ -89,4 +109,177 @@ def test_startup_timer
+  end
+
+  def test_ipv6_echo_rx_interval
+    bg = BfdGlobal.new
+    if validate_property_excluded?('bfd_global', 'ipv6_echo_rx_interval')
+      assert_nil(bg.ipv6_echo_rx_interval)
+      assert_raises(Cisco::UnsupportedError) do
+        bg.ipv6_echo_rx_interval = 100
+      end
+      return
+    end
+    assert_equal(bg.default_ipv6_echo_rx_interval, bg.ipv6_echo_rx_interval)
+    bg.ipv6_echo_rx_interval = 100
+    assert_equal(100, bg.ipv6_echo_rx_interval)
+    bg.ipv6_echo_rx_interval = bg.default_ipv6_echo_rx_interval
+    assert_equal(bg.default_ipv6_echo_rx_interval, bg.ipv6_echo_rx_interval)
+  end
+
+  def test_slow_timer
+    bg = BfdGlobal.new
+    assert_equal(bg.default_slow_timer, bg.slow_timer)
+    bg.slow_timer = 5000
+    assert_equal(5000, bg.slow_timer)
+    bg.slow_timer = bg.default_slow_timer
+    assert_equal(bg.default_slow_timer, bg.slow_timer)
+  end
+
+  def test_ipv4_slow_timer
+    bg = BfdGlobal.new
+    if validate_property_excluded?('bfd_global', 'ipv4_slow_timer')
+      assert_nil(bg.ipv4_slow_timer)
+      assert_raises(Cisco::UnsupportedError) do
+        bg.ipv4_slow_timer = 10_000
+      end
+      return
+    end
+    assert_equal(bg.default_ipv4_slow_timer, bg.ipv4_slow_timer)
+    bg.ipv4_slow_timer = 10_000
+    assert_equal(10_000, bg.ipv4_slow_timer)
+    bg.ipv4_slow_timer = bg.default_ipv4_slow_timer
+    assert_equal(bg.default_ipv4_slow_timer, bg.ipv4_slow_timer)
+  end
+
+  def test_ipv6_slow_timer
+    bg = BfdGlobal.new
+    if validate_property_excluded?('bfd_global', 'ipv6_slow_timer')
+      assert_nil(bg.ipv6_slow_timer)
+      assert_raises(Cisco::UnsupportedError) do
+        bg.ipv6_slow_timer = 25_000
+      end
+      return
+    end
+    assert_equal(bg.default_ipv6_slow_timer, bg.ipv6_slow_timer)
+    bg.ipv6_slow_timer = 25_000
+    assert_equal(25_000, bg.ipv6_slow_timer)
+    bg.ipv6_slow_timer = bg.default_ipv6_slow_timer
+    assert_equal(bg.default_ipv6_slow_timer, bg.ipv6_slow_timer)
+  end
+
+  def test_fabricpath_slow_timer
+    bg = BfdGlobal.new
+    if validate_property_excluded?('bfd_global', 'fabricpath_slow_timer')
+      assert_nil(bg.fabricpath_slow_timer)
+      assert_raises(Cisco::UnsupportedError) do
+        bg.fabricpath_slow_timer = 15_000
+      end
+      return
+    end
+    assert_equal(bg.default_fabricpath_slow_timer, bg.fabricpath_slow_timer)
+    bg.fabricpath_slow_timer = 15_000
+    assert_equal(15_000, bg.fabricpath_slow_timer)
+    bg.fabricpath_slow_timer = bg.default_fabricpath_slow_timer
+    assert_equal(bg.default_fabricpath_slow_timer, bg.fabricpath_slow_timer)
+  end
+
+  def test_interval
+    bg = BfdGlobal.new
+    arr = %w(100 100 25)
+    if validate_property_excluded?('bfd_global', 'interval')
+      assert_nil(bg.interval)
+      assert_raises(Cisco::UnsupportedError) do
+        bg.interval = arr
+      end
+      return
+    end
+    assert_equal(bg.interval, bg.default_interval)
```

Comment:
Vars are swapped. You want to specify *expected* first:

```
-    assert_equal(bg.interval, bg.default_interval)
+    assert_equal(bg.default_interval, bg.interval)
```

Ditto below and also in the ipv4 & ipv6 interval tests.